packages/cozy-client/src/models/dacc.js

```diff
@@ -71,3 +80,45 @@ export const sendMeasureToDACC = async (client, remoteDoctype, measure) => {
     throw error
   }
 }
+
+export const buildAggregateParams = params => {
```

Comment:
jsdoc

Reply:
* @param {object} params - The unformatted DACC aggregate params

Can you add: 
params.measureName 
params.startDate optional 
params.endDate optional

Reply:
I direclty added a typedef